l2-contracts/contracts/AttesterRegistry.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+contract AttesterRegistry {
+    // Owner of the contract (the ConsensusAuthority contract).
+    address public owner;
+    // A map of node owners => attesters (used for attester lookups).
+    mapping(address => Attester) public attesters;
+    // An array to keep track of attester node owners (used for iterating attesters).
+    address[] public attesterOwners;
+    // The current committee list. Weight and public key are stored explicitly
+    // since they might change after committee selection.
+    CommitteeAttester[] public committee;
+
+    struct Attester {
+        // Voting weight.
+        uint256 weight;
+        // ECDSA public key.
+        bytes pubKey;
+        // A flag stating if the attester is inactive. Inactive attesters are not
+        // considered when selecting committees. Only inactive attesters can
+        // be removed from the registry.
+        bool isInactive;
+        // The epoch in which the attester became inactive. We need to store this since
+        // we may want to impose a delay between an attester becoming inactive and
+        // being able to be removed.
+        uint256 inactiveSince;
```

Comment:
this doesn't seem to be updated anywhere

Reply:
Should be removed as part of the inactivity delay mechanism.

Reply:
My bad. Removed now